packages/commercetools/composables/src/getters/checkoutShippingGetters.ts

```diff
@@ -0,0 +1,45 @@
+import { CheckoutShippingGetters } from '@vue-storefront/core';
+import { Address } from './../types/GraphQL';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const getAddresses = (shipping: any, criteria?: Record<string, any>) => [];
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const getMethods = (shipping: any, criteria?: Record<string, any>) => [];
+
+export const getAddressPostCode = (address: Address) => address?.postalCode;
+export const getAddressStreetName = (address: Address) => address?.streetName;
+export const getAddressStreetNumber = (address: Address) => address?.streetNumber;
+export const getAddressCity = (address: Address) => address?.city;
+export const getAddressFirstName = (address: Address) => address?.firstName;
+export const getAddressLastName = (address: Address) => address?.lastName;
+export const getAddressCountry = (address: Address) => address?.country;
+export const getAddressPhone = (address: Address) => address?.phone;
+export const getAddressEmail = (address: Address) => address?.email;
+export const getAddressProvince = (address: Address) => address?.state;
+export const getAddressCompanyName = (address: Address) => address?.company;
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const getAddressTaxNumber = (address: Address) => '';
+export const getAddressApartmentNumber = (address: Address) => address?.apartment;
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const isAddressDefault = (address: Address) => false;
+
+const checkoutShippingGetters: CheckoutShippingGetters<any, any, any> = {
```

Comment:
types

Reply:
We could refactor it just to `<Address>` if we removed 3 method from comment above

Reply:
Considering we will be making changes in getters in the future I don't see a reason to introduce BCs that don't bring any value except refactoring now

Reply:
What kind of BC? It is a new getter and new composable